src/v/redpanda/application.cc

```diff
@@ -664,14 +664,30 @@ void application::wire_up_services() {
           _proxy_client,
           to_yaml(*_proxy_client_config, config::redact_secrets::no))
           .get();
+
+        construct_single_service(_proxy_client_cache);
+
+        using namespace std::literals::chrono_literals;
+
+        auto kafka_api = config::node().kafka_api.value();
+        _proxy_client_cache
+          ->start(
+            smp_service_groups.proxy_smp_sg(),
+            to_yaml(*_proxy_client_config, config::redact_secrets::no),
+            _proxy_config->client_cache_max_size(),
```

Comment:
This commit should instead be in the commit where we add the client cache to proxy.h